packages/suite/src/views/view-only/MacWindow.tsx

```diff
@@ -10,13 +10,21 @@ import { ReactNode } from 'react';
 import styled from 'styled-components';
 
 const Window = styled.div<{ $elevation: Elevation }>`
-    width: 258px;
+    min-width: 258px;
     height: 184px;
     border: 1px solid ${mapElevationToBorder};
     background-color: ${mapElevationToBackground};
     border-bottom: 0;
     border-top-left-radius: ${borders.radii.sm};
     border-top-right-radius: ${borders.radii.sm};
+    flex: 1 1 auto;
+
+    @media (max-width: ${variables.SCREEN_SIZE.SM}) {
+        height: calc(184px + ${spacingsPx.md});
+        border-bottom-left-radius: ${borders.radii.sm};
+        border-bottom-right-radius: ${borders.radii.sm};
```

Comment:
I think you don't have to duplicate these styles, because it's in upper scope

Reply:
This adds bottom radius as in this case the bottom of the MacWindow is not connected with the rest of the content, hence it looks kinda weird without the radius, so I've added it here conditionally, as we don't want it for bigger screens.